visualizer/parseTree.js

```diff
@@ -11,7 +11,6 @@
   }
 })(this, function(ohm, ohmEditor, CheckedEmitter, document, cmUtil, d3, domUtil) {
   var ArrayProto = Array.prototype;
-  function $(sel) { return document.querySelector(sel); }
```

Comment:
Since we use `$` so much in this file, I'd be tempted to put the following at the top of the file:

```js
var $ = domUtil.$;
```